R/compute_duration.R

```diff
@@ -154,16 +154,23 @@ compute_duration <- function(start_date,
   # Checks
   assert_date_vector(start_date)
   assert_date_vector(end_date)
-  assert_character_scalar(in_unit, values = valid_time_units())
+  assert_character_scalar(in_unit, values = c(
```

Comment:
`in_unit` should accept the same values as `out_unit` except for values referring to weeks.